plopfile.js

```diff
@@ -50,7 +69,12 @@ module.exports = (plop) => {
     ],
     actions: (/** @type {InputData} */ { name, platforms }) => {
       const path = require("path");
+
       const exclusive = platforms !== "all";
+      const includeMacOS =
```

Comment:
Shouldn't there be a similar check to see if react-native-windows is installed or not?

Reply:
When we get Windows support, we will have to add it. For now, I think this is sufficient.